src/webgpu/shader/execution/builtin/builtin.ts

```diff
@@ -211,8 +214,14 @@ function toStorage(ty: Type, expr: string): string {
   return expr;
 }
 
+// Currently all values are packed into buffers of 16 byte strides
+const kValueStride = 16;
+
 /**
- * Runs a builtin test
+ * Runs the list of builtin tests, possibly splitting the tests into multiple
+ * dispatches to keep the input data within the buffer binding limits.
+ * run() will pack the scalar test cases into smaller set of vectorized tests
+ * if `cfg.vectorize` is defined.
```

Comment:
nit: Suggest adding:  

> If defined, cfg.vectorize is the vector width, one of 2,3, or 4.

Reply:
There was already a comment on the `Config.vectorize` field, but I've expanded this to be explicit about the allowed values.